src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -954,5 +954,58 @@ string GetPropertyValue(string propertyName)
 
             GetPropertyValue("RootNamespace").Should().Be("Project_Name_With_Spaces");
         }
+
+        [Fact]
+        public void It_errors_on_windows_sdk_assembly_version_conflicts()
+        {
+            var testProjectA = new TestProject()
+            {
+                Name = "ProjA",
+                TargetFrameworks = "net5.0-windows10.0.19041"
+            };
+            testProjectA.SourceFiles.Add("ProjA.cs", @"namespace ProjA
+{
+    public class ProjAClass
+    {
+        public static string str { get { return Windows.Media.Devices.ColorTemperaturePreset.Auto.ToString(); } }
+        public string ProjBstr { get { return ProjB.ProjBClass.str; } }
+    }
+}");
+            var testProjectB = new TestProject()
+            {
+                Name = "ProjB",
+                TargetFrameworks = "net5.0-windows10.0.19041",
+            };
+            testProjectB.SourceFiles.Add("ProjB.cs", @"namespace ProjB
+{
+    public class ProjBClass
+    {
+        public static string str { get { return Windows.Media.Devices.ColorTemperaturePreset.Auto.ToString(); } }
+    }
+}");
+            testProjectA.ReferencedProjects.Add(testProjectB);
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProjectA)
+                .WithProjectChanges((path, project) =>
```

Comment:
Nit: I've recently added a `ProjectChanges` property to the `TestProject` class:

```c#
public List<Action<XDocument>> ProjectChanges { get; } = new List<Action<XDocument>>();
```

This will let you apply changes to a specific project without having to check the filename.

Reply:
Which branch was that change checked into? I'm not able to find it in 5.0.2xx or master.

Reply:
Never mind, looks like it's in #15134, which hasn't been merged yet :-)